src/main/java/com/thirdcc/webapp/service/impl/EventActivityServiceImpl.java

```diff
@@ -42,11 +52,56 @@ public EventActivityServiceImpl(EventActivityRepository eventActivityRepository,
     @Override
     public EventActivityDTO save(EventActivityDTO eventActivityDTO) {
         log.debug("Request to save EventActivity : {}", eventActivityDTO);
+        Set<EventStatus> eventStatuses = new HashSet<EventStatus>() {{
+            add(EventStatus.OPEN);
+            add(EventStatus.POSTPONED);
+        }};
```

Comment:
@kachun333 Can help to explain this part of code? What is the purpose of eventStatuses? And why do we want to store it as HashSet?

Reply:
it was used in 
```
Event event = eventRepository
            .findOneByIdAndStatusIn(eventActivityDTO.getEventId(), eventStatuses)
            .orElseThrow(() -> new BadRequestException("This event does not exists or it is not happening"));
```

The reason I use `Set` instead of `List` is because I think there shouldn't be duplicate EventStatus. Feel Free to use `List` if you want to